os-config.tpl.yml

```diff
@@ -146,6 +146,7 @@ rancher:
       - /usr/bin/ros:/usr/bin/cloud-init-save:ro
       - /usr/bin/ros:/usr/sbin/netconf:ro
       - /usr/bin/ros:/usr/sbin/wait-for-docker:ro
+      - /usr/rancher:/usr/share/rancher:ro
```

Comment:
Wouldn't is be better to put `/usr/rancher` on the host at `/usr/share/rancher`?  It there a specific reason for mapping the folder to a different directory?